src/goto-cc/gcc_mode.cpp

```diff
@@ -1022,6 +1022,9 @@ int gcc_modet::asm_output(
       output_files.begin()->second=="/dev/null"))
     return EX_OK;
 
+  #if 0
+  // The below disturbs some scripts that rummage through assembly
+  // output.
```

Comment:
While this will fix one problem it produces problems elsewhere: there are cases where we need to see that asm output. I'm sure there is a way of placing the output in the file in a way that does not disturb the scripts?